Sources/Datadog/Core/Telemetry.swift

```diff
@@ -41,4 +41,13 @@ extension Telemetry {
     func error(_ message: String, stack: String? = nil) {
         error(message, kind: nil, stack: stack)
     }
+
+    /// Collect execution error.
+    ///
+    /// - Parameters:
+    ///   - message: The error message.
+    ///   - stack: The stack trace.
+    func error(_ message: String, error: Error?) {
+        self.error(message, kind: "swift", stack: String(describing: error))
```

Comment:
I'm open to suggestion on how to report `Swift.Error`. Here, `kind` is `swift` because it's a thrown error from the Swift runtime. The `stack` is of string type with no indication of format, in RUM telemetry the `stack` is described as `The stack trace or the complementary information about the error`.

Reply:
@maxep what about using `DDError` here?

Reply:
Yes I'm on it, sorry I missed this one!